<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogfinance</class>
 <widget class="QMainWindow" name="dialogfinance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 127)</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion des revenus</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>301</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Gestion</string>
      </property>
      <widget class="QPushButton" name="pushButton_updaterev">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>261</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sélectionnez un element dans le tableau</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_sup_rev">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_fich_rev">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>230</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Ficher D'historique</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stat_rev">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>180</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Statistiques</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>301</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Ajouter un revenu</string>
      </property>
      <widget class="QPushButton" name="pushButton_ajouter_rev">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>160</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>66</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Montant</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_7">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>169</width>
         <height>98</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLineEdit" name="lineEdit_type">
          <property name="maxLength">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_id">
          <property name="maxLength">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit_rev"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_montant_rev">
          <property name="maxLength">
           <number>9</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView_rev">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>180</y>
        <width>441</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>441</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Recherche et tri</string>
      </property>
      <widget class="QPushButton" name="pushButton_trie_rev">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>100</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Trier</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_montant_rev">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>70</y>
         <width>111</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Par montant </string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_az_rev">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tri A à Z</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_date_rev">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>50</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Par date</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_supp_rev">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_search_rev">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion depenses</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>301</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Ajouter une depense</string>
      </property>
      <widget class="QPushButton" name="pushButton_ajouter_dp">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>160</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>66</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Catégorie</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Montant</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>169</width>
         <height>98</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLineEdit" name="lineEdit_categorie">
          <property name="maxLength">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_id_dp">
          <property name="maxLength">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit_dp"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_montant_dp">
          <property name="maxLength">
           <number>9</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>301</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Gestion</string>
      </property>
      <widget class="QPushButton" name="pushButton_fich_dp">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>230</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Ficher D'historique</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_update_dp">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sélectionnez un element dans le tableau</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supp_dp">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_stat_dp">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>180</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Statistiques</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView_dp">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>180</y>
        <width>441</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>441</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Recherche et tri</string>
      </property>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_supp_dp">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_search_dp">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_trie_dp">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>100</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Trier</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_date_dp">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>50</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Par date</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_az_dp">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>30</y>
         <width>83</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tri A à Z</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_montant_dp">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>70</y>
         <width>111</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Par montant </string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
